integration-tests/src/test/java/com/datastax/oss/quarkus/CassandraHealthCheckIT.java

```diff
@@ -15,36 +15,27 @@
  */
 package com.datastax.oss.quarkus;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
 
 import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusTest;
-import io.restassured.RestAssured;
-import io.restassured.common.mapper.TypeRef;
-import io.restassured.response.Response;
-import java.util.List;
-import java.util.Map;
 import javax.ws.rs.core.Response.Status;
 import org.junit.jupiter.api.Test;
 
 @QuarkusTest
 @QuarkusTestResource(CassandraTestResource.class)
 public class CassandraHealthCheckIT {
+
   @Test
   public void healthCheckShouldReportStatusUp() {
-    // when
-    Response response = RestAssured.with().get("/health/ready");
-
-    // then
-    assertThat(response.statusCode()).isEqualTo(Status.OK.getStatusCode());
-
-    Map<String, Object> body = response.as(new TypeRef<Map<String, Object>>() {});
-    assertThat(body.get("status")).isEqualTo("UP");
 
-    @SuppressWarnings("unchecked")
-    List<Map<String, Object>> checks = (List<Map<String, Object>>) body.get("checks");
-    assertThat(checks.size()).isOne();
-    assertThat(checks.get(0).get("name"))
-        .isEqualTo("DataStax Apache Cassandra Driver health check");
+    when()
+        .get("/health/ready")
+        .then()
+        .statusCode(Status.OK.getStatusCode())
+        .body("status", equalTo("UP"))
+        .body("checks", hasSize(1))
+        .body("checks[0].name", equalTo("DataStax Apache Cassandra Driver health check"));
```

Comment:
Re-wrote this test (and the other) to better leverage the rest-assured DSL.  It's now more declarative, which in turn should make what it's testing that much clearer.  The DSL explicitly exposes the Hamcrest matchers as part of it's API so this did strongly suggest converting to use those matchers, which I've done here.  I'm well aware that we use assertj in driver code but for a Quarkus integration (more specifically a Quarkus integration for which the hamcrest lib is a first-class citizen and part of the testing API) this seemed like a reasonable change.

Reply:
Sounds very reasonable, and definitely easier to read. 👍